docs/iris/src/userguide/change_management_goals.txt

```diff
@@ -0,0 +1,9 @@
+To reduce code maintenance problems to an absolute minimum, Iris applies
```

Comment:
i recommend removing these sections from the user guide

this section is already in use in the white paper, so the content is preserved.